doc/source/Platforms.rst

```diff
@@ -351,6 +352,16 @@ This instructions are meant to be a reference that users can follow to set up th
    # Note - need to pin to version 5
    brew install qt@5.15.3
 
+.. warning::
+  The :code:`brew install qt@5.15.3` command may not successfully run, in which case you can try :code:`brew install qt@5` instead.
```

Comment:
Would it be better to just say `qt@5` and remove the reference to `qt@5.15.3`?

Reply:
:-) same thought

Reply:
Yes, I agree. Thanks for the suggestion (both of you).